util/util.go

```diff
@@ -233,3 +234,12 @@ func WriteError(w io.Writer, err error, lastError error) error {
 	}
 	return err
 }
+
+// Runtime is the default command to use to run the container.
+func Runtime() string {
+	runtime := os.Getenv("BUILDAH_RUNTIME")
+	if runtime != "" {
+		return runtime
+	}
+	return DefaultRuntime
```

Comment:
I'm not seeing this being set anywhere, only some removals from old code....

Reply:
It is in util/types.go